cylc/flow/unicode_rules.py

```diff
@@ -151,3 +152,18 @@ class XtriggerNameValidator(UnicodeRuleChecker):
     RULES = [
         allowed_characters(r'a-zA-Z0-9', '_')
     ]
+
+
+class MessageTriggerValidator:
+    """Provides a method for validating custom output message trigger
+    contents"""
+
+    @classmethod
+    def validate(cls, string):
+        message = (
+            'a task message cannot contain a colon unless it starts with a '
+            'logging severity level, e.g. "INFO:"')
+        first_part = string.split(':')[0]
+        if first_part == string or first_part in LOG_LEVELS.keys():
+            return (True, None)
+        return (False, message)
```

Comment:
Not exactly a unicode thing, but not sure where else to put it

Reply:
This is very much the right place.